public/week4/list-store.js

```diff
@@ -27,8 +24,8 @@ store.dispatchToken = Dispatcher.register(function(payload){
 			return;
 	}
 
-	store.emitChange();
+	ListStore.emitChange();
 });
 
-return store;
+module.exports = ListStore;
 
```

Comment:
overall it seems to me that the flux architecture is implemented correctly with nice modularization and seperate folders etc i couldn't spot any big issues and the code is really readable